components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1860,15 +1860,12 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
     protected async ensureIsUsageBasedFeatureFlagEnabled(user: User): Promise<void> {
         const teams = await this.teamDB.findTeamsByUser(user.id);
-        const isUsageBasedBillingEnabled = await getExperimentsClient().getValueAsync(
+        const isUsageBasedBillingEnabled = await getExperimentsClientForBackend().getValueAsync(
             "isUsageBasedBillingEnabled",
             false,
             {
-                identifier: user.id,
```

Comment:
What is the difference here? :thinking: 

Is this a free-form shape we :100: control?

Reply:
(asking because the former shape looks like it followed a pattern/external guideline/external interface.

Reply:
User ID, Email and Region are first class, The rest are custom attributes we control